app/library/postgres.py

```diff
@@ -224,6 +224,25 @@ async def get_scenario_by_friendly_name_from_postgres(
 
 
 async def delete_scenario_from_postgres(scenario_id: str, session):
+    # Get scene first
+    scenario_obj: Scenario = get_scenario(scenario_id, session)
+    if scenario_obj is None:
+        raise ValueError("Scenario ID not valid")
+
+    # Delete the screenshot of scenario, if it has one
+    if scenario_obj.screenshot_url:
```

Comment:
IMO `screenshot_url` is a weird name for this. It makes sense for the scene since we are taking a screenshot of the scene and storing it, but in this case it's not really a screenshot. `display_image_url` or something along those lines might be better.